docs/index.md

```diff
@@ -1,6 +1,6 @@
 <h1><img src="/images/logo.svg" alt="logo" width="450px" style="display: block; margin-left: auto; margin-right: auto"/></h1>
 
-Larq is an open-source deep learning library for training neural networks with extremely low-precision weights and activations, such as Binarized Neural Networks (BNNs).
+Larq is an open-source deep learning library for training neural networks with extremely low-precision weights and activations, such as Binarized Neural Networks (BNNs)[^1].
```

Comment:
Alternatively we could just add a link here.

Reply:
The reference currently isn't clickable (nor does it show the URL from the bib), would be nice if it is (especially for newcomers to BNNs)

Reply:
Good point 👍 

I added a link in c98459e1a755fde6187e3b332922c88aa29982df